bin/internal/run-zowe.sh

```diff
@@ -36,7 +36,11 @@ export ROOT_DIR=$(cd $(dirname $0)/../../;pwd)
 launch_components_list=$(${ROOT_DIR}/bin/internal/get-launch-components.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}")
 . ${ROOT_DIR}/bin/internal/prepare-workspace.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}" -t "${launch_components_list}"
 
-# FIXME: Zowe Launcher can take responsibility from here
-for component_id in $(echo "${launch_components_list}" | sed "s/,/ /g"); do
-  . ${ROOT_DIR}/bin/internal/start-component.sh -c "${INSTANCE_DIR}" -r "${ROOT_DIR}" -o "${component_id}" &
-done
+if [ "${USE_LAUNCHER}" = "true" ]
+then
+  ${ROOT_DIR}/components/launcher/bin/zowe_launcher "${launch_components_list}"
```

Comment:
The launcher isn't a component but it starts components. There should be a different location for zowe launcher. Perhaps ${ROOT_DIR}/bin/launcher or similar.